src/coreComponents/linearAlgebra/interfaces/trilinos/EpetraVector.cpp

```diff
@@ -127,10 +120,10 @@ void EpetraVector::create( arrayView1d< real64 const > const & localValues,
   Epetra_Map const map( LvArray::integerConversion< long long >( -1 ),
                         localSize,
                         0,
-                        Epetra_MpiComm( MPI_PARAM( comm ) ) );
-  m_vector = std::make_unique< Epetra_FEVector >( Copy,
+                        EpetraComm( MPI_PARAM( comm ) ) );
+  m_vector = std::make_unique< Epetra_FEVector >( View,
```

Comment:
Does this mean that the `localValues` vector has to _survive_ at least as the `EpetraVector`?

Reply:
Yes. This is the shallow copy construction, I implemented it first in Tpetra and then decided to change this one as well. It works with the way we currently use this constructor, where it wraps around an `array1d< real64 > m_localRhs`. We may want to think of two different constructors though.